app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -29,7 +33,7 @@ def proxy_command_options(template_invocation, host)
             ),
             :name => host.name,
             :check_mode => host.host_param('ansible_roles_check_mode')
-          )
+          ).merge(proxy_command_provider_inputs(template_invocation))
```

Comment:
This could also probably be generalized and done in the REX PR, right? :thinking: